src/Chatkit.php

```diff
@@ -299,6 +301,71 @@ public function deleteRoom($options)
 
         return $this->execCurl($ch);
     }
+    
+    /**
+     * Get all read cursors for a user
+     *
+     * @param array $options
+     *              [Available Options]
+     *              • user_id (string|required): Represents the ID of the user that you want to get the rooms for.
+     * @return array
+     * @throws ChatkitException or MissingArgumentException
+     */
+    public function getUserReadCursors($options)
+    {
+        if (is_null($options['user_id'])) {
+            throw new MissingArgumentException('You must provide the ID of the user that you want to get the cursors for');
+        }
+
+        $user_id = $options['user_id'];
+
+        $ch = $this->createCurl(
+            $this->cursor_settings,
+            "/cursors/0/users/$user_id",
+            $this->getServerToken([ 'user_id' => $user_id ]),
+            'GET'
+        );
+
+        return $this->execCurl($ch);
+    }
+
+    /**
+     * Sets the read cursor for a user in a room.
+     *
+     * @param array $options
+     *              [Available Options]
+     *              • user_id (string|required): Represents the ID of the user that you want to get the rooms for.
+     *              • room_id (string|required): Represents the ID of the user that you want to get the rooms for.
+     *              • position (string|required): Represents the ID of the user that you want to get the rooms for.
```

Comment:
I _believe_ `position` needs to be an integer, but maybe the server is being "forgiving" and allowing strings as well? Or maybe this is just needs a documentation tweak 😊